helm/values.yaml

```diff
@@ -222,6 +221,28 @@ mcp-servers:
 llm-service:
   enabled: true
 
+# Mock ServiceNow Server Configuration (for testing and development)
```

Comment:
It would be good if the mock pod was only deployed if mocking was enabled, maybe we can check if the URL is set to the mock URL and only enable if that is the case?